File: internal/languages/ruby/testdata/scope/scope.rb
scope_cursor(params[:oops])
scope_cursor(params[:ok] + x)
scope_cursor(x ? params[:oops] : y)
scope_cursor(params[:ok] ? x : y)

scope_nested(params[:oops])
scope_nested(params[:oops] + x)
scope_nested(x ? params[:oops] : y)
scope_nested(params[:oops] ? x : y)

scope_result(params[:oops])
scope_result(params[:oops] + x)
scope_result(x ? params[:oops] : y)
scope_result(params[:ok] ? x : y)

File: internal/languages/javascript/testdata/scope/scope.js
scopeCursor(req.params.oops)
scopeCursor(req.params.ok + x)
scopeCursor(x ? req.params.oops : y)
scopeCursor(req.params.ok ? x : y)

scopeNested(req.params.oops)
scopeResult(req.params.oops + x)
scopeNested(x ? req.params.oops : y)
scopeNested(req.params.oops ? x : y)

scopeResult(req.params.oops)
scopeResult(req.params.oops + x)
scopeResult(x ? req.params.oops : y)
scopeResult(req.params.ok ? x : y)
